packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -630,14 +599,16 @@ describeNoCompat("GC inactive nodes tests", (getTestObjectProvider) => {
 						},
 						{ [LoaderHeader.version]: summaryVersion1 },
 					);
-					const defaultDataStoreContainer2 = await requestFluidObject<ITestDataObject>(
-						container2,
-						"/",
-					);
+					const defaultDataStoreContainer2 =
+						(await container2.getEntryPoint()) as ITestDataObject;
 					defaultDataStoreContainer2._root.set("mode", "write");
 					await waitForContainerWriteModeConnectionWrite(container2);
 
-					const response = await containerRuntime_resolveHandle(container2, {
+					const entryPoint = (await container2.getEntryPoint()) as ITestDataObject;
```

Comment:
nit: You can use `defaultDataStoreContainer2` here instead of getting the entry point again.